rudder-lang/libs/oslib.rl

```diff
@@ -30,6 +30,32 @@ items in unix {
   *
 }
 
+# == AIX
+items in aix {
+  aix_5,
+  aix_6,
+  aix_7,
+  *
+}
+
+# TODO should we make it extendable ?
```

Comment:
The general question, is for all OS, should we add a * (extensible list) in os subversion, given that there won't be anymore such version. 
I think we should not have it